src/ToolBox/SOS/NETCore/SymbolReader.cs

```diff
@@ -126,6 +126,27 @@ public override void Write(byte[] buffer, int offset, int count)
         }
 
         /// <summary>
+        /// Quick fix for Path.GetFileName which incorrectly handles Windows-style paths on Linux
+        /// </summary>
+        /// <param name="pathName"> File path to be processed </param>
+        /// <returns>Last component of path</returns>
+        private static string GetFileName(string pathName)
+        {
+            int pos = -1;
+            for (int i = pathName.Length - 1; i >= 0; --i)
+            {
+                if (pathName[i] == '/' || pathName[i] == '\\')
```

Comment:
You could use pathName.IndexOfAny(new char[] { '/', '\\'}) instead of this code.